docs/Using-Fleet/MDM-migration-guide.md

```diff
@@ -27,7 +27,14 @@ This section provides instructions for migrating your hosts away from your old M
     2. In ABM, assign these hosts' MDM server to Fleet: In ABM, select **Devices** and then select **All Devices**. Then, select **Edit** next to **Edit MDM Server**, select **Assign to the following MDM:**, select your Fleet server in the dropdown, and select **Continue**.
 5. In your old MDM solution, unenroll the hosts to be migrated. MacOS does not allow multiple MDMs to be installed at once.
 6. Send [these guided instructions](#instructions-for-end-users) to your end users to complete the final few steps via Fleet Desktop.
-    * Note that there will be a gap in MDM coverage between when the host is unenrolled from the old MDM and when the host turns on MDM in Fleet.
+    * Note that there will be a gap in MDM coverage between when the host is unenrolled from the old
+      MDM and when the host turns on MDM in Fleet.
+
```

Comment:
Hmm, I think we're missing some context about the End user migration workflow feature here. The dark/light logo is a specific feature for this workflow.

I think we should add a ### title like "End user migration workflow" above this section. I think we should also add a blurb about why you'd want to use the end user migration workflow and how it works.

We can fill in other aspects of the workflow (voluntary v. forced mode, webhook URL) in a separate PR.